keyboards/keychron/q2/ansi_encoder/rules.mk

```diff
@@ -8,7 +8,7 @@ BOOTLOADER = stm32-dfu
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = no        # Mouse keys
+MOUSEKEY_ENABLE = yes       # Mouse keys
```

Comment:
The keymaps `keychron`, `VIA` or `default` don't seem to use any mouse key... Is this necessary?

Reply:
I think it is necessary since someone may want to add mouse key through via client.

Reply:
Also, if somebody wants to compile with the QMK Configurator, it uses these defaults.  And there is no (simple) way to change that.  It's why we tend to prefer that extra keys and mouse keys be enabled by default (since they appear in the configurator), if there is room for it.